packages/code-jumpers/src/jumpers/jumper.ts

```diff
@@ -91,7 +91,7 @@ export abstract class CodeJumper {
 
   private protectFromAccidentalEditing(document_widget: IDocumentWidget) {
     let editor_widget = document_widget as IDocumentWidget<FileEditor>;
-    editor_widget.title.label = editor_widget.title.label + ' (external)';
+    editor_widget.title.label = editor_widget.title.label + '';
```

Comment:
I would suggest to replace it with a comment altogether:

```suggestion
// We used to adjust `editor_widget.title.label` here but an upstream
// bug (https://github.com/jupyterlab/jupyterlab/issues/10856) prevents
// us from doing so anymore.
```

Reply:
Yes, better that way